libs/application-generic/src/webhooks/usecases/send-webhook-message/send-webhook-message.usecase.ts

```diff
@@ -0,0 +1,83 @@
+import { Inject, Injectable, Scope } from '@nestjs/common';
+import { Svix } from 'svix';
+import shortid from 'shortid';
+
+import { PinoLogger } from 'nestjs-pino';
+import { EnvironmentRepository } from '@novu/dal';
+import { SendWebhookMessageCommand } from './send-webhook-message.command';
+import { WrapperDto } from '../../dtos/webhook-payload.dto';
+
+const LOG_CONTEXT = 'SendWebhookMessageUseCase';
+
+@Injectable()
+export class SendWebhookMessage {
+  constructor(
+    @Inject('SVIX_CLIENT') private svix: Svix,
+    private logger: PinoLogger,
+    private environmentRepository: EnvironmentRepository
+  ) {
+    this.logger.setContext(LOG_CONTEXT);
```

Comment:
not a huge one, just for consistency. 
```suggestion
    this.logger.setContext(this.constructor.name);
```